src/cljs/rems/actions/add_licenses.cljs

```diff
@@ -0,0 +1,116 @@
+(ns rems.actions.add-licenses
+  (:require [clojure.string :as str]
+            [re-frame.core :as rf]
+            [rems.actions.action :refer [action-button action-form-view action-comment button-wrapper collapse-action-form]]
+            [rems.autocomplete :as autocomplete]
+            [rems.status-modal :as status-modal]
+            [rems.text :refer [text]]
+            [rems.util :refer [fetch post!]]))
+
+(rf/reg-fx
+ ::fetch-licenses
+ (fn [[user on-success]]
+   (fetch "/api/licenses"
+          {:handler on-success
+           :headers {"x-rems-user-id" (:eppn user)}})))
+
+(rf/reg-event-fx
+ ::open-form
+ (fn
+   [{:keys [db]} _]
+   {:db (assoc db
+               ::comment ""
+               ::potential-licenses #{}
+               ::selected-licenses #{})
+    ::fetch-licenses [(get-in db [:identity :user])
+                      #(rf/dispatch [::set-potential-licenses %])]}))
+
+(defn- assoc-search-term
+  "Prepopulate a search term with localized names and unlocalized title"
+  [license]
+  (assoc license :search-term (str/join "" (conj (mapcat :title (vals (:localizations license)))
```

Comment:
I think the name `:search-term` is a bit misleading... how about `:all-titles`

Reply:
Renamed